lib/permit_ecto/resolver.ex

```diff
@@ -17,6 +17,39 @@ defmodule Permit.Ecto.Resolver do
   import Permit.Helpers, only: [resource_module_from_resource: 1]
 
   @impl Permit.ResolverBase
+  def resolve(
+        subject,
+        authorization_module,
+        resource_module,
+        action,
+        %{use_loader?: true} = meta,
```

Comment:
It could be possible to not have to use an additional `use_loader?` option if `permit_phoenix/controller.ex` allows differentiating between nil and a function at plug level. (line 365)